lib/device/library/windows/eject.rb

```diff
@@ -0,0 +1,163 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2013 whiteleaf. All rights reserved.
+#
+# rubocop:disable Style/MethodName
+# rubocop:disable Style/VariableName
+
+class Device
+  module Library
+    module Windows
+      FSCTL_LOCK_VOLUME = 0x00090018
+      FSCTL_DISMOUNT_VOLUME = 0x00090020
+      IOCTL_STORAGE_MEDIA_REMOVAL = 0x002d4804
+      IOCTL_STORAGE_EJECT_MEDIA = 0x002d4808
+      FILE_SHARE_READ = 0x00000001
+      FILE_SHARE_WRITE = 0x00000002
+      OPEN_EXISTING = 3
+      INVALID_HANDLE_VALUE = -1
+      GENERIC_READ = 0x80000000
+      GENERIC_WRITE = 0x40000000
+      DRIVE_REMOVABLE = 2
+      DRIVE_CDROM = 5
+      NULL = 0
+      WIN_FALSE = 0
+      LOCK_TIMEOUT = 10.0 # Seconds
+      LOCK_RETRIES = 20
+
+      def eject(volume_name)
+        device_root = get_device_root_dir(volume_name)
+        unless ejectVolume(device_root)
+          raise Device::CantEject, "Cannot eject"
+        end
+      end
+
+      private
+
+      def openVolume(cDriveLetter)
```

Comment:
__[reek]__
* [UncommunicativeMethodName] has the name 'openVolume' [:ref](https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Method-Name.md)
* [UncommunicativeParameterName] has the parameter name 'cDriveLetter' [:ref](https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Parameter-Name.md)
* [UtilityFunction] doesn't depend on instance state (maybe move it to another class?) [:ref](https://github.com/troessner/reek/blob/master/docs/Utility-Function.md)